.env

```diff
@@ -26,6 +26,6 @@ TESTS_DATA=tests/_data
 TESTS_SUPPORT=tests/_support
 TESTS_ENVS=tests/_envs
 
-CORE_BRANCH=develop
+CORE_VERSION=v1.3.3
```

Comment:
I'm thinking this should default to the latest release: 

https://docs.github.com/en/github/administering-a-repository/linking-to-releases

Then have the option to checkout a specific release based on the version number, but always default to latest?

Reply:
Yes, I had this thought but gave up spending much time on it when saw it's one path to get the latest release and another to get by version/tag release. Slight difference, but enough.
https://github.com/wp-graphql/wp-graphql/releases/latest/download/wp-graphql.zip
https://github.com/wp-graphql/wp-graphql/releases/download/v1.3.3/wp-graphql.zip

I can add some logic to handle that when 'latest'.

Reply:
Or, maybe we just go with 'latest' and move on. I'm overthinking this.

Reply:
And if that's the case, should I just use wp-cli to get it from wordpress.org?

Reply:
Ya, I mean we probably will want the ability to test against multiple versions for CI with a matrix of some sort, so it's good to handle both. 

I think folks working locally will want the latest, but the option to override.

Reply:
Ok, gonna put part of this back.